### PR TITLE
Fix firefox/choose url config.

### PR DESCRIPTION
**Pull request by pmclanahan**

### CODE REVIEW

File: bedrock/firefox/urls.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
         views.all_downloads, name='firefox.all'),
     page('firefox/channel', 'firefox/channel.html'),
     redirect('^firefox/channel/android/$', 'firefox.channel', locale_prefix=False),
-    url('firefox/choose', views.choose, name='firefox.choose'),
+    url(r'^firefox/choose/$', views.choose, name='firefox.choose'),
```
Comment (jgmize) [inline bedrock/firefox/urls.py]: should we make the trailing slash optional?

Comment (pmclanahan) [inline bedrock/firefox/urls.py]: trailing slashes aren't optional in urls.py files. it is optional in global.conf.

Comment (pmclanahan) [inline bedrock/firefox/urls.py]: django fixes them for us if the version w/o the slash isn't found.

**Comment by pmclanahan** [comment]
@ckprice it fixes it. it's not working in some situations, and it's wrong in the code. if you'd like us to not fix it then we'll hold off.

**Pull request merged**